### DISCUSSION TITLE
'Simulator' object has no attribute 'generate_interface'

### DISCUSSION THREAD

**Discussion by Berlin-Law-code** [caltech-netlab/gym-acnportal#10] [Q&A]
Dear author: in the process of running, reported an error, can you help me to answer it?  
File "D:\The third\structure learning\gym-acnportal-master\tutorials\lessonx_training_running_rl_agent.py", line 169, in interface_generating_function
    return sim.generate_interface(GymTrainingInterface)
AttributeError: 'Simulator' object has no attribute 'generate_interface'

**Comment by sunash** [comment]
This is likely due to using an old version of acnportal. What version of acnportal are you using?

If you're interested, there is another set of environments for RL by others in this group that is under more active development: https://github.com/chrisyeh96/sustaingym
